app/mailers/task_mailer.rb

```diff
@@ -0,0 +1,34 @@
+class TaskMailer < ApplicationMailer
+  def owner_task_created(project, task)
+    @project = project
+    @task = task
+    @owner = ProjectMembership.find_by(project: @project, role: :owner).user
+
+    mail(to: owner.email.pluck(:email))
+  end
+
+  def member_task_created(project, task)
+    @project = project
+    @task = task
+
+    for user in project.users do
```

Comment:
Better to code it like
```suggestion
    project.users.each do |user|
```